amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/ExchangeContainer.java

```diff
@@ -57,18 +57,68 @@ public static void putExchange(NamespaceName namespaceName, String exchangeName,
         });
     }
 
-    public static AmqpExchange getExchange(NamespaceName namespaceName, String exchangeName) {
-        if (namespaceName == null || StringUtils.isEmpty(exchangeName)) {
-            return null;
+    public CompletableFuture<AmqpExchange> asyncGetExchange(PulsarService pulsarService,
```

Comment:
Can this method be protected, or default visibility?

Reply:
This method is used in many places, and I think the public is suitable.